src/status_im/ui/screens/browser/site_blocked/views.cljs

```diff
@@ -21,14 +16,16 @@
     [vector-icons/icon :main-icons/info {:color colors/red}]
     [react/text {:style styles/title-text}
      (i18n/label :t/browsing-site-blocked-title)]
-    [react/text {:style styles/description-text}
-     (i18n/label :t/browsing-site-blocked-description1)
-     [chat-link]
-     [react/text (i18n/label :t/browsing-site-blocked-description2)]]
+    [react/view {:align-items :center}
+     [react/text {:style styles/description-text} (i18n/label :t/browsing-site-blocked-description1)]
+     [react/touchable-highlight {:on-press #(.openURL react/linking "status-im://chat/public/status")}
```

Comment:
I think you can still have the link creation in a function just check how nested-text component works, here you probably just want the style to be returned with the on-press attribute. However I am not 100% sure it is supported though it is worth testing I think it is done somewhere else

Reply:
nested-text didn't work so i decided to go this way, just to fix this for release, and it's not great that `react/text` crashes the app, should be handled properly for sure

Reply:
It crashed the app because it was code that never got reached so far. This is a leftover from typography pr that got missed, nested test has to be explicit.

How did you do the nested-text component? It takes strings or tuples of style+string as inputs

Reply:
https://github.com/status-im/status-react/blob/496f3f1cc78703c3d629f9f458a0e966b04e5c61/src/status_im/ui/screens/privacy_policy/views.cljs#L8

It works I've done it

Reply:
@yenda  yeah that works, updated